bgpd/bgp_route.c

```diff
@@ -4515,7 +4515,8 @@ int bgp_nlri_parse_ip(struct peer *peer, struct attr *attr,
 			if (pnt + BGP_ADDPATH_ID_LEN >= lim)
 				return BGP_NLRI_PARSE_ERROR_PACKET_OVERFLOW;
 
-			addpath_id = ntohl(*((uint32_t *)pnt));
```

Comment:
The same code exist at bgp_nlri_parse_label, bgp_nlri_parse_evpn, bgp_nlri_parse_vpn as well. Should we just make it a macro and use at all places?